checkfiles.py

```diff
@@ -1002,11 +1002,16 @@ def fetch_files(session, url, search_query, out, include_unexpired_upload=False,
         }
         errors = job['errors']
         item_url = urljoin(url, job['@id'])
-
+        fileObject = session.get(item_url)
         r = session.get(item_url + '@@upload?datastore=database')
         if r.ok:
```

Comment:
Need to make sure the fileObject is ok too:
```suggestion
        if fileObject.ok and r.ok:
```